The typescript-prettier dependencies are missing eslint-config-prettier, eslint-plugin-prettier and prettier

The current command for installing dependencies for a "typescript-prettier" configuration looks like this:
> npm install --save-dev eslint-config-react-app @typescript-eslint/eslint-plugin@^4.0.0 @typescript-eslint/parser@^4.0.0 babel-eslint@^10.0.0 eslint@^7.5.0 eslint-plugin-flowtype@^5.2.0 eslint-plugin-import@^2.22.0 eslint-plugin-jsx-a11y@^6.3.1 eslint-plugin-react@^7.20.3 eslint-plugin-react-hooks@^4.0.8

But when I run eslint I get the errors below.

Installing these packages fixes it:
- `eslint-config-prettier`
- `eslint-plugin-prettier`
- `prettier`

Were these omitted on purpose or is this a bug? Are specific versions needed?

Thanks


```
$ eslint --ext js,ts,tsx src

Oops! Something went wrong! :(

ESLint: 7.10.0

ESLint couldn't find the config "prettier/@typescript-eslint" to extend from. Please check that the name of the config is correct.

The config "prettier/@typescript-eslint" was referenced from the config file in "/Users/user/project/.eslintrc.js".

If you still have problems, please stop by https://eslint.org/chat/help to chat with the team.

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```


```
$ eslint --ext js,ts,tsx src

Oops! Something went wrong! :(

ESLint: 7.10.0

ESLint couldn't find the plugin "eslint-plugin-prettier".

(The package "eslint-plugin-prettier" was not found when loaded as a Node module from the directory "/Users/user/project".)

It's likely that the plugin isn't installed correctly. Try reinstalling by running the following:

    npm install eslint-plugin-prettier@latest --save-dev

The plugin "eslint-plugin-prettier" was referenced from the config file in ".eslintrc.js".

If you still can't figure out the problem, please stop by https://eslint.org/chat/help to chat with the team.

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

```
$ eslint --ext js,ts,tsx src

Oops! Something went wrong! :(

ESLint: 7.10.0

Error: Cannot find module 'prettier'
Require stack:
- /Users/user/project/node_modules/eslint-plugin-prettier/eslint-plugin-prettier.js
- /Users/user/project/node_modules/eslint/lib/cli-engine/config-array-factory.js
- /Users/user/project/node_modules/eslint/lib/cli-engine/cascading-config-array-factory.js
- /Users/user/project/node_modules/eslint/lib/cli-engine/cli-engine.js
- /Users/user/project/node_modules/eslint/lib/eslint/eslint.js
- /Users/user/project/node_modules/eslint/lib/eslint/index.js
- /Users/user/project/node_modules/eslint/lib/cli.js
- /Users/user/project/node_modules/eslint/bin/eslint.js
Occurred while linting /Users/user/project/Api.ts:2
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:980:15)
    at Function.Module._load (internal/modules/cjs/loader.js:862:27)
    at Module.require (internal/modules/cjs/loader.js:1040:19)
    at require (/Users/user/project/node_modules/v8-compile-cache/v8-compile-cache.js:161:20)
    at Program (/Users/user/project/node_modules/eslint-plugin-prettier/eslint-plugin-prettier.js:168:26)
    at /Users/user/project/node_modules/eslint/lib/linter/safe-emitter.js:45:58
    at Array.forEach (<anonymous>)
    at Object.emit (/Users/user/project/node_modules/eslint/lib/linter/safe-emitter.js:45:38)
    at NodeEventGenerator.applySelector (/Users/user/project/node_modules/eslint/lib/linter/node-event-generator.js:254:26)
    at NodeEventGenerator.applySelectors (/Users/user/project/node_modules/eslint/lib/linter/node-event-generator.js:283:22)
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

--- a/pages/cra/typescript-prettier.tsx
+++ b/pages/cra/typescript-prettier.tsx
@@ -9,7 +9,7 @@ export default function PageCreateReactAppTypeScriptPrettier() {
       <PageHeader>TypeScript &amp; Prettier</PageHeader>
       <Section title="Dependencies">
         <CodeBlock isScrollable>
-          {`npm install --save-dev eslint-config-react-app @typescript-eslint/eslint-plugin@^4.0.0 @typescript-eslint/parser@^4.0.0 babel-eslint@^10.0.0 eslint@^7.5.0 eslint-plugin-flowtype@^5.2.0 eslint-plugin-import@^2.22.0 eslint-plugin-jsx-a11y@^6.3.1 eslint-plugin-react@^7.20.3 eslint-plugin-react-hooks@^4.0.8`}
+          {`npm install --save-dev eslint-config-react-app @typescript-eslint/eslint-plugin@^4.0.0 @typescript-eslint/parser@^4.0.0 babel-eslint@^10.0.0 eslint@^7.5.0 eslint-plugin-flowtype@^5.2.0 eslint-plugin-import@^2.22.0 eslint-plugin-jsx-a11y@^6.3.1 eslint-plugin-react@^7.20.3 eslint-plugin-react-hooks@^4.0.8 prettier@^2.1.2 eslint-config-prettier@^6.12.0 eslint-plugin-prettier@^3.1.4`}
         </CodeBlock>
       </Section>
       <Section title=".eslintrc.js">

